mlrun/serving/v2_serving.py

```diff
@@ -404,11 +404,12 @@ def base_data(self):
         return base_data
 
     def push(self, start, request, resp=None, op=None, error=None):
+        start_str = start.strftime("%Y-%m-%d %H:%M:%S.%f %z")
```

Comment:
@assaf758 ,because of (2) I added the space because we assuming in the monitor stream that the timezone have 6 char.
I can change it to 5 and delete the space if you think it better?

Reply:
seems that datetime.isoformat() gives us just what we want:
```
>>> n=datetime.datetime(2023, 3, 22, 13, 0, 18, 0, tzinfo=datetime.timezone(datetime.timedelta(seconds=3600)))
>>> n.isoformat(sep=' ',timespec='microseconds')
'2023-03-22 13:00:18.000000+01:00'
```